integrations/git_test.go

```diff
@@ -268,15 +296,17 @@ func ensureAnonymousClone(t *testing.T, u *url.URL) {
 }
 
 func lockTest(t *testing.T, remote, repoPath string) {
-	_, err := git.NewCommand("remote").AddArguments("set-url", "origin", remote).RunInDir(repoPath) //TODO add test ssh git-lfs-creds
```

Comment:
So we ignored the `remote` parameter?

Reply:
It's not necessary - the remote parameter is set already when we clone.

This line is incorrect for SSH cloned repos

Reply:
I should say that this line is what was hiding the SSH locking bug from being detected. It's incorrectly overriding the remote that is being set on clone .